### ISSUE TITLE
Uploaded progress gets reset after closing the app

### ISSUE THREAD

**Issue by officiallor** [XITRIX/iTorrent#367]
Let's assume that I have uploaded 2GB after the torrent was completed. This means that I downloaded for example 1GB and uploaded 2GB so ratio is 2.0. 

When I restart the client the statistics are going back to 0. It's like per session (app launch) instead of persistent statistics. In this way you don't know what is your ratio.

Another issue is that it counts the traffic with peers (probably from connections initialization-control traffic) as downloaded after the restart, so after downloading 1GB and restarting I see more KB being downloaded. It doesn't affect anything serious but I wanted to mention it.

Tested on 2.0.14-1 iOS 18

**Comment by XITRIX** [comment]
True, I forgot to store statistics on disk, I remember I had some problem saving it, and decided to implement it later (and completely forgot about it)

Need to finally implement it ...
